script.js

```diff
@@ -7,6 +7,7 @@ function addNewIntern() {
   // Get values from the form
   let title = document.getElementById("internTitleF").value;
   let info = document.getElementById("internInfoF").value;
+  console.log("intern info:" , info);
```

Comment:
I think we don't need this line.

Reply:
@Kishan-Ved ,
i removed that line.

thanks!